app/views/reimbursement_requests/_form_attachments.html.erb

```diff
@@ -11,6 +11,17 @@
       </div>
     </div>
     <div class="form-row">
+      <% if @reimbursement_request.children.count > 0 %>
```

Comment:
anywhere you were using `@reimbursement_request.children` it should be `@reimbursement_request.attachments`